utils/fetchGitDiff.js

```diff
@@ -3,7 +3,7 @@ const { execChildProcess } = require("./common");
 module.exports = function fetchGitDiff( baseBranch = "master" ) {
 
   // git command to pull out the changed file names between current branch and base branch (Excluded delelted files which cannot be fetched now)
-  let command = `git diff --relative --name-only --diff-filter=d ${baseBranch}...HEAD`;
+  let command = `git diff --relative --name-only --diff-filter=d ${baseBranch}`;
```

Comment:
I couldn't get the motive here. Can you shed some light here?

Reply:
IMO, it should. Can you explain it in more detail for further clarification?

Reply:
I have tried a few use cases with `...HEAD`, it is working as intended for me.

Reply:
I think this PR is good to go after resolving this discussion 🔜 .  Please make sure that the PR doesn't have any conflicts with the master 😉